### ISSUE TITLE
[FEAT] Display reset times

### ISSUE THREAD

**Issue by Brunhine** [Brunhine/ffxivtodo#12]
Only suggestion would be more for newer players: including times and day of week (weeklies) for reset. It’s helpful to know that jumbo cactpot is Saturday nights, Fashion is Fri-Tues AM, etc.

https://www.reddit.com/r/ffxiv/comments/ktgkjd/ffxiv_todo_tracker/gim54zm?utm_source=share&utm_medium=web2x&context=3

**Comment by Karpfador** [comment]
Not only should the times be displayed, those weeklies should have their own countdown instead of being reset with the normal weekly reset.

**Comment by Brunhine** [comment]
> Not only should the times be displayed, those weeklies should have their own countdown instead of being reset with the normal weekly reset.

I agree. However, I have yet to decide how I want to display those separate timers for those items just yet.
